pkg/cmd/server/bootstrappolicy/namespace_policy_test.go

```diff
@@ -0,0 +1,46 @@
+package bootstrappolicy
+
+import (
+	"strings"
+	"testing"
+
+	"k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac/bootstrappolicy"
+)
+
+func TestOpenshiftNamespacePolicyNamespaces(t *testing.T) {
```

Comment:
I made this fatal in #16517 just like upstream.  Do you still want the tests?

Reply:
> I made this fatal in #16517 just like upstream. Do you still want the tests?

don't care